contrib/forms/example2/print.php

```diff
@@ -68,9 +68,9 @@
 <span class="title"><?php echo xlt($form_name); ?></span><br />
 
 <!-- container for the main body of the form -->
-<div id="print_form_container">
+<div id="print-form-container">
```

Comment:
I am guessing this is going to be a very prevalent issue in OpenEMR's codebase. Is there a chance of css breakage when using underscore instead of a dash?
(if not and this is, as i suspect, a very prevalent issue in OpenEMR's codebase, then would consider ignoring this rule since the extensive fixes required could open the door to lots of bugs)

Reply:
Indeed, it's a common issue. Currently, we have almost 700 occurrences of it. The goal of this change to conform to the default pattern of the `selector-id-pattern` rule, which is kebab-case. But according to the [MDN docs for the id attribute](https://developer.mozilla.org/en-US/docs/Web/HTML/Global_attributes/id), there's no issue in using `_` for ids. Since both kebab-case and snake_case ids are very common patterns in this project, it makes sense to disable this rule. I'll update the config. Thanks!